Cargo.toml

```diff
@@ -19,3 +19,12 @@ exclude = [
   "benches/payout-stakers",
   "bin/cliain",
 ]
+
+[profile.release]
+# Substrate runtime requires unwinding.
+panic = "unwind"
+
+[profile.production]
+inherits = "release"
+lto = true
+codegen-units = 1
```

Comment:
On a quick look, I could not find in offical LLVM docs https://llvm.org/docs/CodeGenerator.html that indeed setting this to one can improve performance. Shall we back up on it till next task, unless some benches have been made that prove performance improvement with that setting enabled?

Reply:
E.g. this says it does not help much https://github.com/paritytech/polkadot/issues/4311#issuecomment-972403714

Reply:
After discussing offline, no harm to include this